docs/DevGuide/CreatingExecutables.md

```diff
@@ -0,0 +1,93 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Creating Executables {#dev_guide_creating_executables}
+
+There are several different types of executables that can be built:
+- An executable that uses Charm++ to run in parallel,
+  e.g. `Executables/ParallelInfo`
+- An executable that does not use Charm++, does not run in parallel, and
+  supplies its own `main`, e.g. `Executables/Benchmark`
+- An executable that uses Charm++ to run in parallel but supplies its own `main`
+- An executable that uses custom compilation or linking flags,
+  e.g. `DebugPreprocessor`
+- Executables used for evolutions or elliptic solves
+
+### Executable Using Charm++ for Parallelization
+
+### Executable Using Charm++ with Custom main()
```

Comment:
Is there supposed to be something between these headers?  If not, should they be different levels?

Reply:
Larry will add this in #751, I added a comment explaining this in the file so nobody gets confused.